src/plots/frame_attributes.js

```diff
@@ -39,13 +39,15 @@ module.exports = {
     },
     data: {
         valType: 'any',
+        role: 'object',
         description: [
             'A list of traces this frame modifies. The format is identical to the',
             'normal trace definition.'
         ].join(' ')
     },
     layout: {
         valType: 'any',
+        role: 'object',
```

Comment:
Is it correct?

Reply:
I think no, for both of these. According to the logic we discussed the other day, `role` should not be given when there's a `valType`, and that applies even to `valType: 'any'`.

Reply:
Fixed by https://github.com/plotly/plotly.js/pull/5432/commits/a363aecf61ea49d67c3e3f74baa949e7e3e9f458.